l10n_it_account/models/account_tax.py

```diff
@@ -205,7 +205,7 @@ def _compute_totals_tax(self, data):
                     continue
 
                 tax_balance += child_balance
-                if child.account_id:
+                if child.tax_exigibility == "on_invoice":
```

Comment:
Sulla base di come `account_id` è stato migrato in OpenUpgrade (rif. [link](https://github.com/OCA/OpenUpgrade/blob/420ac0f79fa0cc216500d73df60741b4618a304b/addons/account/migrations/13.0.1.1/openupgrade_analysis_work.txt#L455-L461)) ritengo che per rendere il controllo più conforme all’originale sia opportuno controllare:

`if account_ids` dove:

```
            account_ids = (
                child.mapped("invoice_repartition_line_ids.account_id")
                | child.mapped("refund_repartition_line_ids.account_id")
            ).ids

Cosa ne pensi?

Reply:
@AlessandroSkab hai visto https://github.com/OCA/l10n-italy/pull/2799#discussion_r874799014 ?